src/Math.php

```diff
@@ -52,6 +52,9 @@ public static function ceilFloat($n) : int{
 	 * @return float[]
 	 */
 	public static function solveQuadratic(float $a, float $b, float $c) : array{
+		if((int) $a === 0){
+			throw new \Exception("Coefficient a cannot be 0!");
```

Comment:
be specific. This should throw an invalid argument exception.

Reply:
Fixed.